### DISCUSSION TITLE
Generic validation errors, or at least an array of strings.

### DISCUSSION THREAD

**Discussion by freddi301** [TanStack/form#979] [Ideas]
What I am now doing is to JSON.stringify the validation errors structure I need to JSON.parse it later.
This erases type information tough.

**Comment by Balastrong** [comment]
Can you please share a snippet or an example on stackblitz of what you're currently doing and what solution you're looking for? :)

**Reply by freddi301** [reply]
https://codesandbox.io/p/sandbox/qzrn8q

it is simplified example, but the requirement is to display diverse types of visual feedbacks about errors.
These feedback includes boolean flags to display error icons or red color in borders, list of textual error messages and many more.

What was the reason behind the design choice to allow only strings as validation errors?
